src/terra-dev-site/guide/Internationalization/BuildingComponentsWhichIncludeTranslations.e.guide.md

```diff
@@ -0,0 +1,114 @@
+# Building Components Which Include Translations
+
+Below is an example of a component that renders a button with a hard-coded string.
+
+```jsx
+import React from 'react';
+
+const Toggle = () => (
+  <button type="button">Toggle</button>
+);
+```
+
+There are various ways to render the string as a translation with react-intl. We recommend using [react-intl's FormattedMessage component](https://github.com/yahoo/react-intl/wiki/Components#formattedmessage). Below is an example highlighting the use react-intl's formatted message component.
+
+```jsx
+import React from 'react';
+import Base from 'terra-base';
+import { FormattedMessage } from 'react-intl';
+
+const Toggle = () => (
+  <button type="button">
+    <FormattedMessage id="App.toggle" />
+  </button>
+);
+
+const App = () => (
+  <Base className="custom-App" locale="en">
+    <Toggle />
+  </Base>
+);
+
+export default App;
+```
+
+**Note:** *The terra-base component doesn't need to wrap individual components with translations, it should be used once to wrap the entire React app.*
+
+By default `<FormattedMessage>` will render the formatted string into a `<span>`. If you need to customize rendering, we recommend using the render prop version of FormattedMessage.
```

Comment:
might be helpful to add a quick link reference here
> render prop version of FormattedMessage.

Reply:
Updated in this commit: https://github.com/cerner/terra-ui/pull/132/commits/cc0c8713331225234ca6d889d6cca8802fbc4d3c